src/pages/ServiceGraph/ServiceGraphPage.tsx

```diff
@@ -124,6 +126,12 @@ export default class ServiceGraphPage extends React.Component<
     }
   };
 
+  handleReady = (cy: any) => {
+    if (cy !== undefined) {
```

Comment:
Remember our friend **falsey**?
Falsey gives us a cleaner syntax of `if(!cy) {`

Reply:
@mtho11 falsey?

Reply:
d'oh, right, should just be:   if (cy)